src/Arcus.Templates.Worker.Messaging/Program.cs

```diff
@@ -77,9 +77,15 @@ public static IHostBuilder CreateHostBuilder(string[] args)
 #endif
                        .ConfigureServices((hostContext, services) =>
                        {
+#if ServiceBusQueue
                            services.AddServiceBusQueueMessagePump(secretProvider => secretProvider.GetRawSecretAsync("ARCUS_SERVICEBUS_CONNECTIONSTRING"))
                                    .WithServiceBusMessageHandler<EmptyMessageHandler, EmptyMessage>();
-                           
+#endif
+#if ServiceBusTopic
+                               services.AddServiceBusTopicMessagePump("Receive-All", secretProvider => secretProvider.GetRawSecretAsync("ARCUS_SERVICEBUS_CONNECTIONSTRING"))
+                                       .WithServiceBusMessageHandler<EmptyMessageHandler, EmptyMessage>();
```

Comment:
Most important change related to this issue: a single `Program.cs` file where all the messaging source options can be set. We can add EventHubs later to this setup.